angular/projects/spark-angular/src/lib/components/inputs/Percentage.stories.ts

```diff
@@ -55,12 +56,130 @@ const modules = {
 };
 
 export const percentageInput = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-input-container>
+      <div class="
+        sprk-b-InputContainer__icon-container
+        sprk-b-InputContainer__icon-container--narrow"
+      >
+        <label sprkLabel>Percentage</label>
+        <sprk-icon
+          iconName="percent"
+          additionalClasses="sprk-b-InputContainer__icon sprk-b-InputContainer__icon--right"
+          sprk-input-icon
+        ></sprk-icon>
+        <input
+          class="sprk-b-InputContainer__input--has-icon-right"
+          name="percentage"
+          type="tel"
+          sprkInput
+        />
+      </div>
+    </sprk-input-container>
+  `,
+});
+
+percentageInput.story = {
+  name: 'Default',
+  parameters: {
+    jest: [
+      'sprk-input-container.component',
+      'sprk-input.directive',
+      'sprk-label.directive',
+    ],
+  },
+};
+
+export const invalidPercentageInput = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-input-container>
+      <div class="
+        sprk-b-InputContainer__icon-container
+        sprk-b-InputContainer__icon-container--narrow"
+      >      
+        <label sprkLabel>Percentage</label>
+        <sprk-icon
+          iconName="percent"
+          additionalClasses="sprk-b-InputContainer__icon sprk-b-InputContainer__icon--right"
+          aria-invalid="true"
+          sprk-input-icon
+        ></sprk-icon>
+        <input
+          class="sprk-b-InputContainer__input--has-icon-right sprk-b-TextInput--error"
+          name="percentage"
+          type="tel"
+          sprkInput
+        />
+      </div>
+      <span sprkFieldError>
+        <sprk-icon
+          iconName="exclamation-filled"
```

Comment:
Check into missing aria-hidden here

Reply:
Established that we can't add this in Angular because it isn't an Input on the `sprk-icon` component. Resolving this.